molecule/provisioner/lint/ansible_lint.py

```diff
@@ -54,6 +54,28 @@ class AnsibleLint(base.Base):
 
     The `x` option has to be passed like this due to a `bug`_ in Ansible Lint.
 
+    An additional option is to use the following option for ansible-lint on `molecule.yml`
+    
+    .. code-block:: yaml
+    
+        provisioner:
```

Comment:
Is the indentation right here (and on line 42), I would have expected one space to need to be removed